sass/main.scss

```diff
@@ -160,28 +163,28 @@ table {
   margin-bottom:30px;
   width:100%;
   text-align:left;
-  color:#3f3f3f;
+  color: var(--table-color);
   border-collapse:collapse;
-  border:1px solid #e8e8e8
+  border: 1px solid var(--border-color);
 }
 table tr:nth-child(even) {
-  background-color:#f7f7f7
+  background-color: var(--table-row-alt-color);
 }
 table th,
 table td {
   padding:10px 15px
 }
 table th {
-  background-color:#f0f0f0;
-  border:1px solid #dedede;
-  border-bottom-color:#c9c9c9
+  background-color: var(--table-row-header-color);
+  border: 1px solid var(--table-head-border-color);
+  border-bottom-color: var(--table-head-border-bottom-color);
 }
 table td {
-  border:1px solid #e8e8e8
+  border: 1px solid var(--border-color);
 }
 .site-header {
-  border-top:5px solid #424242;
-  border-bottom:1px solid #e8e8e8;
+  border-top: 5px solid var(--header-color);
```

Comment:
IMO, this is quite ugly in dark mode now, and the page looks better without it. The bottom border is fine.
I'm not too sure what purpose the grey bar at the top of the page has, anyway. As a temporary fix, we could just hide the top bar in dark mode? Or maybe there's a different colour we could use (`#bdbdbd` as the mirror of `#424242`)?

Reply:
Yeah, I think I'll just remove that border.